packages/test/test-end-to-end-tests/src/test/gc/gcTrailingOps.spec.ts

```diff
@@ -0,0 +1,177 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import {
+    ITestObjectProvider,
+    waitForContainerConnection,
+    createSummarizer,
+    summarizeNow,
+    ITestContainerConfig,
+    mockConfigProvider,
+} from "@fluidframework/test-utils";
+import { describeNoCompat, ITestDataObject, itExpects, TestDataObjectType } from "@fluidframework/test-version-utils";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import { stringToBuffer } from "@fluidframework/common-utils";
+import { ISummaryTree } from "@fluidframework/protocol-definitions";
+import { IGCRuntimeOptions } from "@fluidframework/container-runtime";
+import { getGCStateFromSummary } from "./gcTestSummaryUtils";
+import { defaultGCConfig } from "./gcTestConfigs";
+
+describeNoCompat("GC trailing ops tests", (getTestObjectProvider) => {
+    const tests = (tombstoneEnabled: boolean = false) => {
+        let provider: ITestObjectProvider;
+
+        const sweepTimeoutMs = 0;
```

Comment:
Consider setting this to a non-zero value adding a delay of this value if `tombstoneEnabled` is true before loading second summarizer. I think that way it it a little closer to real scenarios.

Reply:
I thought so as well, if we add basically no delay, we should be able to test this with all drivers. That was the sacrifice I was making. Also, it may be hard to read from the tests, but some tests do not run with sweepTimeoutMs at all so the unreference scenario is tested as well. Adding delay of 1ms.